flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveTableSource.java

```diff
@@ -79,7 +78,8 @@
 
 	private static final Logger LOG = LoggerFactory.getLogger(HiveTableSource.class);
 
-	private final JobConf jobConf;
+	private final JobConf hiveConf;
```

Comment:
Let's not change the name to `hiveConf` since it's not an instance of `HiveConf`

Reply:
OK, revert to `jobConf`